packages/components/components/elvis-chip/CHANGELOG.json

```diff
@@ -2,6 +2,22 @@
   "$schema": "../../changelogSchema.json",
   "name": "elvis-chip",
   "content": [
+    {
+      "date": "18.06.24",
+      "version": "2.6.0",
+      "changelog": [
+        {
+          "type": "new_feature",
+          "changes": [
+            "The legend type of chip now supports a new prop: <code>markerStyle</code>. This can be <code>dot</code>, <code>line</code>, or <code>dashed</code>."
```

Comment:
```suggestion
            "The legend type of chip now supports a new prop: <code>markerStyle</code>. This can be <code>dot</code> (default), <code>line</code>, or <code>dashed</code>."
```